feature-find-meal/src/main/java/com/kinandcarta/feature/find/meal/view/FindMealFragment.kt

```diff
@@ -92,13 +92,14 @@ class FindMealFragment : Fragment() {
     }
 
     private fun onReservedMeal(state: FindMealViewModel.State.ReservedMeal) {
-        mealsAdapter.notifyItemChanged(state.position)
```

Comment:
Realised this wasn't working as I had anticipated.  Due to no update actually being carried out on the list.

Reply:
I think what you've changed it to is cleaner anyway, it would just be nice not to have to worry about the position at all and maybe let the adapter figure that out (which I think ListAdapter does). Not a problem for now though, since this works

Reply:
Have converted over to use the ListAdapter 👍